.github/workflows/ci.yml

```
@@ -495,6 +495,43 @@ jobs:
         with:
           job-name: Account Console IT
 
+  webauthn-integration-tests:
+    name: WebAuthn IT
+    runs-on: ubuntu-latest
+    needs: build
+    timeout-minutes: 75
```

I'd suggest we try with 30 minutes for now. If these tests end up taking 75 that really is way to long.

```suggestion
    timeout-minutes: 30
```